WcaOnRails/app/models/post.rb

```diff
@@ -34,7 +34,7 @@ def self.crash_course_post
   end
 
   def deletable
-    !is_crash_course_post?
+    !new_record? && !is_crash_course_post?
```

Comment:
I think this has 30% lower cognitive load

`!(new_record? || is_crash_course_post?)`